src/Sulu/Bundle/AudienceTargetingBundle/Tests/Unit/Rule/TargetGroupEvaluatorTest.php

```diff
@@ -99,7 +97,7 @@ public function testEvaluate(
         $this->assertEquals($evaluatedTargetGroup, $this->targetGroupEvaluator->evaluate($frequency, $currentTargetGroup));
     }
 
-    public static function provideEvaluationData()
+    public static function provideEvaluationData(): array
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes